answerking_app/tests/test_categories.py

```diff
@@ -477,15 +525,17 @@ def test_put_add_duplicated_item_in_url_to_category_return_400(self):
             content_type="application/json",
         )
         actual = response.json()
-        error_msg: ErrorMessage = {
-            "error": {
-                "message": "Resource update failure",
-                "details": "Item already in category",
-            }
+        expected: DetailError = {
+            "detail": "A server error occurred.",
+            "status": 400,
+            "title": "A server error occurred.",
+            "type": "http://testserver/problems/error/",
```

Comment:
I feel like this error message is a bit vague. Is there anyway we can tell they user that the name already exists?

Reply:
maybe put it in the detail field?